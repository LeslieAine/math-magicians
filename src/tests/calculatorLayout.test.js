import renderer from 'react-test-renderer';
import CalculatorLayout from '../components/CalculatorLayout';

it('renders correctly', () => {
  const tree = renderer.create(<CalculatorLayout />).toJSON();
  expect(tree).toMatchSnapshot();
});
