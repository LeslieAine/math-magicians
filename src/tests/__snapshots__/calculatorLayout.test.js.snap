// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="calculatorLayout"
>
  <h1
    className="calculatorLayoutTitle"
  >
    Let us do some math!
  </h1>
  <div
    className="calculator"
  >
    <div
      className="calculator-display"
      data-testid="display"
    >
      0
    </div>
    <div
      className="calculator-buttons"
    >
      <button
        className="calculator-button"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="calculator-button"
        onClick={[Function]}
        type="button"
      >
         +/- 
      </button>
      <button
        className="calculator-button"
        onClick={[Function]}
        type="button"
      >
         % 
      </button>
      <button
        className="calculator-button right"
        onClick={[Function]}
        type="button"
      >
         รท 
      </button>
      <button
        className="calculator-button"
        onClick={[Function]}
        type="button"
      >
         7 
      </button>
      <button
        className="calculator-button"
        onClick={[Function]}
        type="button"
      >
         8 
      </button>
      <button
        className="calculator-button"
        onClick={[Function]}
        type="button"
      >
         9 
      </button>
      <button
        className="calculator-button right"
        onClick={[Function]}
        type="button"
      >
         x 
      </button>
      <button
        className="calculator-button"
        onClick={[Function]}
        type="button"
      >
         4 
      </button>
      <button
        className="calculator-button"
        onClick={[Function]}
        type="button"
      >
         5 
      </button>
      <button
        className="calculator-button"
        onClick={[Function]}
        type="button"
      >
         6 
      </button>
      <button
        className="calculator-button right"
        onClick={[Function]}
        type="button"
      >
         - 
      </button>
      <button
        className="calculator-button"
        onClick={[Function]}
        type="button"
      >
         1 
      </button>
      <button
        className="calculator-button"
        onClick={[Function]}
        type="button"
      >
         2 
      </button>
      <button
        className="calculator-button"
        onClick={[Function]}
        type="button"
      >
         3 
      </button>
      <button
        className="calculator-button right"
        onClick={[Function]}
        type="button"
      >
         + 
      </button>
      <button
        className="calculator-button"
        id="zero"
        onClick={[Function]}
        type="button"
      >
         0 
      </button>
      <button
        className="calculator-button"
        onClick={[Function]}
        type="button"
      >
         . 
      </button>
      <button
        className="calculator-button right"
        onClick={[Function]}
        type="button"
      >
         = 
      </button>
    </div>
  </div>
</div>
`;
